frontend/src/pages/RollingpaperPage/components/MessageTextArea.tsx

```diff
@@ -47,7 +48,7 @@ export const MessageTextArea = ({
       >
         {children}
       </StyledTextArea>
-      <StyledTextLength>{value?.toString.length}/500</StyledTextLength>
+      <StyledTextLength>{value?.toString().length}/500</StyledTextLength>
```

Comment:
상단에서 타입을 string으로 줬으니 `{value.length}`로 수정하면 어떨까요?

```suggestion
      <StyledTextLength>{value.length}/500</StyledTextLength>
```